package.json

```diff
@@ -23,7 +23,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint --fix --ext .js,.jsx ./src/."
   },
   "eslintConfig": {
```

Comment:
you can move extends body to .eslintrc.json file and remove "eslintConfig" from package.json

Reply:
alright , removed

Reply:
I can see that you removed it instead of moving to .eslintrc didn't it break anything? did you try to run lint after it?

Reply:
yep